docs/conf.py

```diff
@@ -314,6 +323,84 @@ def find_source():
     ),
 ]
 
+
+def _make_estimator_overview(app):
+    """Make estimator overview table."""
+    import pandas as pd
+
+    from aeon.registry import all_estimators
+
+    def _does_not_start_with_underscore(input_string):
+        return not input_string.startswith("_")
+
+    # Columns for the output table
+    COLNAMES = ["Estimator name", "Module", "Family of method"]
```

Comment:
Personal preference:
```suggestion
    COLNAMES = ["Estimator name", "Module", "Method family"]
```
(also see below)